com.gluonhq.scenebuilder.yml

```diff
@@ -24,6 +24,7 @@ modules:
     - install -Dm644 com.gluonhq.scenebuilder.metainfo.xml -t /app/share/metainfo/
     - install -Dm755 scenebuilder.sh /app/bin/SceneBuilder
     - install -Dm644 com.gluonhq.scenebuilder.256.png /app/share/icons/hicolor/256x256/apps/com.gluonhq.scenebuilder.png
+    - install -Dm644 com.gluonhq.scenebuilder.128.png /app/share/icons/hicolor/128x128/apps/com.gluonhq.scenebuilder.png
     - sed -i '/^Categories=/ d' /app/lib/scenebuilder/lib/scenebuilder-SceneBuilder.desktop 
```

Comment:
use `destkop-file-edit` instead of sed

Reply:
And you'll need to add at the top `rename-desktop-file: scenebuilder-SceneBuilder.desktop`